airship.cabal

```diff
@@ -47,6 +48,7 @@ library
                       , http-media
                       , http-types == 0.8.*
                       , lifted-base == 0.2.*
+                      , microlens
```

Comment:
What's your stance on version bounds?

We tend to have `maj.min.*` by default just to limit the fallout from breaking upstream changes, but I know it can be religious discussion and I'm not trying to start one. :)